### PR TITLE
ZD-39493

### PR DESCRIPTION
**Pull request by carlosrodlop**
These two scripts have been generated in response of ZD-39493

@reviewbybees I am not able to save the changes for any of those scripts

### CODE REVIEW

File: add-templates2folders.groovy [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    if (allowedTypes.size() > 0) {
+        jenkinsFolders.each{ folder->
+            //filter example
+            if ((folder.url).equals(pFolders)){
+                // println "[DEBUG]: "+folder.url
+                folder.properties.each{ prop ->
+                    if ((prop instanceof SubItemFilterProperty)==true){
+                        prop = new SubItemFilterProperty (allowedTypes)
+                        if (!processingFlag) processingFlag = true
+                        jenkins.save()
```
Comment (carlosrodlop) [inline add-templates2folders.groovy]: The changes are not saved. Which method should I use for serialization at this point to make changes permanent?

File: create-group4template.groovy [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    g2create = new Group (gc2update, pGroup)
+    //Adding members
+    g2create.setMembers(pUsers)
+    def List<RoleAssignment> assignments = new ArrayList<RoleAssignment>()
+    def rolAssigmment = new RoleAssignment(pRol)
+    assignments.add(rolAssigmment)
+    //Adding roles Group#setRoleAssignments(List<RoleAssignment> roleAssignments)
+    g2create.setRoleAssignments(assignments)
+    g2create.save()
+    jenkins.save()
```
Comment (carlosrodlop) [inline create-group4template.groovy]: The changes are not saved. Which method should I use for serialization at this point to make changes permanent?

**Comment by kuisathaverat** [comment]
check this test have a very useful code related with RBAC 
https://github.com/cloudbees/nectar-rbac-plugin/blob/e56acd58078ca8e243f56c352df4193e69a6a7f1/src/test/java/nectar/plugins/rbac/groups/BaseCommandTest.java

**Comment by carlosrodlop** [comment]
To review:
- https://github.com/cloudbees/nectar-rbac-plugin/blob/e56acd58078ca8e243f56c352df4193e69a6a7f1/src/test/java/nectar/plugins/rbac/groups/BaseCommandTest.java
- https://github.com/cloudbees/nectar-rbac-plugin/blob/e56acd58078ca8e243f56c352df4193e69a6a7f1/src/main/java/nectar/plugins/rbac/groups/GroupContainer.java#L64
- https://github.com/cloudbees/nectar-rbac-plugin/blob/e56acd58078ca8e243f56c352df4193e69a6a7f1/src/main/java/nectar/plugins/rbac/groups/Group.java#L1428

**Comment by carlosrodlop** [comment]
Ready to be merged this first version after testing. 

They have been provided to the customer in ZD-39252. If anything need to be fixed after their feedback, it would be included.

**Pull request merged**